internal/cmd/egctl/translate.go

```diff
@@ -262,9 +262,10 @@ func translateGatewayAPIToGatewayAPI(resources *gatewayapi.Resources) (gatewayap
 
 	// Translate from Gateway API to Xds IR
 	gTranslator := &gatewayapi.Translator{
-		GatewayControllerName:  egv1alpha1.GatewayControllerName,
-		GatewayClassName:       v1beta1.ObjectName(resources.GatewayClass.Name),
-		GlobalRateLimitEnabled: true,
+		GatewayControllerName:   egv1alpha1.GatewayControllerName,
+		GatewayClassName:        v1beta1.ObjectName(resources.GatewayClass.Name),
+		GlobalRateLimitEnabled:  true,
+		EndpointRoutingDisabled: true,
```

Comment:
I understand that we want the routing behavior to be backward compatible, but I think using endpoints as default to route traffic allows EG to bypass kube-proxy's overhead and fully leverage Envoy's advanced capabilities: more LB algorithms, session sticky, etc.

Reply:
I have disabled this in `egctl x translate`, because if we enable it, it will be mandatory, and all translations will fail, I was planning on raising a separate GH issue on dealing with this convenience / correctness dilemma